tests/sentry/api/endpoints/test_organization_transaction_anomaly_detection.py

```diff
@@ -0,0 +1,176 @@
+from collections import namedtuple
+from datetime import datetime, timezone
+from unittest import mock
+
+from django.http import HttpResponse
+from django.urls import reverse
+from freezegun import freeze_time
+
+from sentry.api.endpoints.organization_transaction_anomaly_detection import map_snuba_queries
+from sentry.testutils import APITestCase, SnubaTestCase
+
+
+class OrganizationTransactionAnomalyDetectionEndpoint(APITestCase, SnubaTestCase):
+    endpoint = "sentry-api-0-organization-transaction-anomaly-detection"
+
+    def setUp(self):
+        super().setUp()
+        self.project = self.create_project(organization=self.organization)
+        self.login_as(user=self.user)
+        self.features = {}
+        self.snuba_raw_data = [(0, 0), (3, 0), (0, 0)]
+
+    @mock.patch("sentry.api.endpoints.organization_transaction_anomaly_detection.timeseries_query")
+    @mock.patch("sentry.api.endpoints.organization_transaction_anomaly_detection.get_anomalies")
+    def test_get_start_end(self, mock_get_anomalies, mock_timeseries_query):
+        SnubaTSResult = namedtuple("SnubaTSResult", "data")
+        mock_timeseries_query.return_value = SnubaTSResult(
+            data={"data": self.snuba_raw_data},
+        )
+        mock_get_anomalies.return_value = HttpResponse({"key": "value"})
+        request = {
+            "project": self.project.id,
+            "query": "transaction.duration",
```

Comment:
Could we put a valid query here? Without a field prefix this is actually becoming
"message:transaction.duration"

maybe something like
```suggestion
            "query": "transaction.duration:>5s",
```